content/videos/building-resilient-test-automation-playwright.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Building Resilient Test Automation with Playwright
+date: 2025-01-12
+description: Discover how to build resilient and maintainable test automation frameworks using Playwright. This talk covers strategies for handling flaky tests, implementing effective waiting mechanisms, and creating tests that adapt to changing UI requirements.
+video: 0CDGPWjajcE
+tags: [playwright, automation, testing, resilience]
+image: https://res.cloudinary.com/debsobrien/image/upload/v1640019149/debbie.codes/blog/playwright-automation_h7xm3p.jpg
```

Comment:
remove tags of automation and resilience
the description is terrible see if you can find the correct title and description from the video url